src/main/java/com/synopsys/integration/alert/web/scheduling/SchedulingDescriptor.java

```diff
@@ -21,14 +21,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package com.synopsys.integration.alert.web.controller;
+package com.synopsys.integration.alert.web.scheduling;
```

Comment:
I dont believe this class belongs in the package com.synopsys.integration.alert.web